### PR TITLE
Feat/1945 back out axios

### LINKED ISSUE #1945: As tech lead, I need all frontend app buttons in prod env to point to correct endpoints
**Description:**
As of July 2022, the frontend and backend in the TDP prod environment are located in different domains. the frontend is in an acf domain, while the backend is in a cloudgov domain. The dev and staging env have not been configured to use multiple domains, so we need to ensure that all frontend buttons in prod hit the correct endpoints. 

currently when : 
```
BASE_URL: https://tdp-backend-prod.app.cloud.gov/v1
FRONTEND_BASE_URL: https://tanfdata.acf.hhs.gov
```
- the `request access` button at [tanfdata.acf.hhs.gov/home](tanfdata.acf.hhs.gov/home) does not work. 
- the `submit data files` button at [tanfdata.acf.hhs.gov/data-files](tanfdata.acf.hhs.gov/data-files) does not work. and returns a `403 error`
- the `sign out` button for users with an acf email address redirects to and acf-ams page that returns a  `Invalid redirect uri` and does not successfully logout on first attempt. (_this button functions properly for non-acf email addresses._)

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] request access button at tanfdata.acf.hhs.gov/home functions properly. 
- [ ] users with appropriate permissions can submit data files via submit data file button
- [ ] all acf and non-acf users can sign out properly from frontend
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] reproduce issues in description
- [ ] fix buttons
- [ ] Run Testing Checklist and confirm all tests pass

**Notes:**
- [swagger](https://tdp-backend-prod.app.cloud.gov/swagger/) was used to submit data files to ensure that download button on frontend was functioning properly. [common.py](https://github.com/ADPennington/TANF-app/commits/master) needed changes to facilitate this. 
- frontend sign out button

### PR DESCRIPTION
**Pull request by andrew-jameson**
## Summary of Changes
Pull request closes #1945 

## How to Test
N/A

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
- [x] request access button at tanfdata.acf.hhs.gov/home functions properly. 
- [x] users with appropriate permissions can submit data files via submit data file button
- [x] all acf and non-acf users can sign out properly from frontend
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary
+ [x] **`lfrohlich`** and/or **`adpennington`**  confirmed that ACs are met.

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
  + [x] If this PR introduces backend code changes, are they meaningfully tested?
  + [x] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [x] Frontend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)
  + [x] Backend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)

+ [x] Are backend code style checks passing on CircleCI?
+ [x] Are frontend code style checks passing on CircleCI?
+ [x] Are code maintainability principles being followed?

### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

+ [x] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)

+ [x] Does this PR provide background for why coding decisions were made?
+ [ ] If this PR introduces backend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces frontend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?
+ [x] Can reviewer explain and take ownership of these elements presented in this code review?

### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)

+ [x] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any new security issues?
+ [ ] If new issues detected, is investigation and/or remediation plan documented? 

### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
+ [ ] the purpose of the research
+ [ ] methods used to conduct the research 
+ [ ] who participated in the research
+ [ ] what was tested and how
+ [ ] impact of research on TDP
+ [ ] (_if applicable_) final design mockups produced for TDP development

### CODE REVIEW

File: tdrs-frontend/src/components/GovBanner/GovBanner.jsx [hunk 1]
```diff
@@ -26,28 +26,21 @@ function GovBanner() {
                   switch (process.env.NODE_ENV) {
                     case 'development':
                       return ' Development Demo '
-                      break
                     case 'test':
                       return ' Test Demo '
-                      break
```
Comment (andrew-jameson) [inline tdrs-frontend/src/components/GovBanner/GovBanner.jsx:32]: These were introducing a runtime linter failure, hence their removal. We will not re-add these.

@riatzukiza

File: .circleci/config.yml [hunk 2]
```diff
@@ -718,10 +718,12 @@ workflows:
             branches:
               only:
                 - master
+                - feat/1945-back-out-axios
```
Comment (andrew-jameson) [inline .circleci/config.yml:721]: These need to be backed out before merging.

**Review by ADPennington — APPROVED** [review]
🚀

File: tdrs-frontend/nginx/nginx.conf [hunk 3]
```diff
@@ -2,5 +2,6 @@ server {
     listen 80;
     root /usr/share/nginx/html;
     include locations.conf;
-    add_header Content-Security-Policy "default-src 'none'; script-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self'; connect-src 'self'; manifest-src 'self'; object-src 'none'; frame-ancestors 'none'; form-action 'none';";
+    add_header Content-Security-Policy "default-src 'self'; *.acf.hhs.gov; script-src 'self'; style-src 'self' 'unsafe-inline'; img-src 'self' data:; font-src 'self'; connect-src 'self'; *.acf.hhs.gov manifest-src 'self'; object-src 'none'; frame-ancestors 'none'; form-action 'none';";
+    add_header Access-Control-Allow-Origin "https://tanfdata.acf.hhs.gov"
```
Comment (andrew-jameson) [inline tdrs-frontend/nginx/nginx.conf:6]:
```suggestion
    add_header Access-Control-Allow-Origin "https://tanfdata.acf.hhs.gov";
```

**Pull request merged**